src/utils/hooks/useAnimationFrame.ts

```diff
@@ -25,6 +25,7 @@
  
      const start = useCallback((...args) => {
          cancelAnimationFrame(id.current);
+         //@ts-ignore
```

Comment:
got an error here. when i tried to fix it, my component did not work anymore. any idea to fix that besides ignoring?